assets/dev/scss/frontend/general/_button.scss

```diff
@@ -83,19 +91,32 @@
 
 	span {
 		text-decoration: inherit; //fix for conflict with text-decoration & inline-block
+		align-self: center;
+	}
+
+	&:hover {
+		.elementor-button-icon {
+			i {
+				color: var(--button-icon-color-hover);
+				transition: all 0.3s;
+			}
+		
+			svg {
+				fill: var(--button-icon-color-hover);
+				transition: all 0.3s;
+			}
+		}
 	}
 }
 
 .elementor-element.elementor-button- {
 	@include button-types( '.elementor-button' );
-
 }
 
 .elementor-widget-button {
 	.elementor-button {
 		.elementor-button- {
-
```

Comment:
Please restore the original version.

Reply:
I added the line like the original version

Reply:
You added tabs.

Please check your work on GitHub.
![Uploading Screenshot 2024-05-02 at 13.30.27.png…]()

Reply:
I removed those tabs

Reply:
![Uploading Screenshot 2024-05-02 at 13.50.44.png…]()